papers/Vvmax/calc_vvmax.py

```diff
@@ -0,0 +1,764 @@
+"""
+Script to calculate V and Vmax for CRAFT FRBs
```

Comment:
is the code generic enough to be ueseful for other experiments?

if so, all the more reason to have this exposed in the main repo

Reply:
Yep, this is the part that's been moved. At least, the core of it has.